rdflib/plugins/sparql/parser.py

```diff
@@ -1479,7 +1479,7 @@ def expandCollection(terms):
     "DescribeQuery",
     Keyword("DESCRIBE")
     + (OneOrMore(ParamList("var", VarOrIri)) | "*")
-    + Param("datasetClause", ZeroOrMore(DatasetClause))
+    + ZeroOrMore(ParamList("datasetClause", DatasetClause))
```

Comment:
Does this make a meaningful difference? In most other cases it is as it was before, and the code that processes this remains unchanged as far as I can tell, so I would rather keep it the same unless there is a good reason to change it, which there may be, but I can't quite tell. If there is a good reason to change it then I think a comment would maybe be warranted so that others that come later can easily figure out why DESCRIBE is different from say SELECT in this regard.

Reply:
Two points about this change:
- This does make a meaningful difference; if left the old way, the `datasetClause` attribute would be a single object if one `FROM` or `FROM NAMED` clause is present in the query, which would cause `evalQuery` to fail [here](https://github.com/RDFLib/rdflib/blob/63755b8330cd6ce7650130732cb13a4712131944/rdflib/plugins/sparql/evaluate.py#L642). The updated way would be a list of one element in that case which would allow `evalQuery` to work as intended.
- If you look [here](https://github.com/RDFLib/rdflib/blob/63755b8330cd6ce7650130732cb13a4712131944/rdflib/plugins/sparql/parser.py#L1431), you'll see that I simply copied what was already present for `SELECT` queries to use for `DESCRIBE`; `DESCRIBE` used to be different, but now it actually matches `SELECT`.